spt/OrangeBox/modules/ClientDLL.cpp

```diff
@@ -415,6 +421,43 @@ void ClientDLL::ResetAfterframesQueue()
 
 void ClientDLL::OnFrame()
 {
+#if !defined(OE)
+	extern int GetEntityCount();
+	extern CBaseEntity* GetEntity(int);
+
+	if (y_spt_piwsave.GetString()[0] != '\0')
+	{
+		auto ply = GetEntity(1);
+		if (ply)
+		{
+			auto pphys = ply->VPhysicsGetObject();
+			if (pphys && (pphys->GetGameFlags() & FVPHYSICS_PENETRATING) == 0)
+			{
+				int count = GetEntityCount();
+				for (int i = 0; i < count; ++i)
```

Comment:
I might well be wrong here, but if worldspawn is 0 and the player is 1, isn't it unnecessary to start this loop at index 0, and you could use 2 instead? Not that it'd make a big difference anyway.